### PR TITLE
React Native

### PR DESCRIPTION
**Pull request by vyacheslav-pushkin**

### CODE REVIEW

File: packages/front-generator/src/generators/react-native/app/template/components/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+    this.props
+      .onLoginSubmit(this.login, this.password)
+      .then(_value => {
+        return AsyncStorage.setItem(REST_TOKEN_STORAGE_KEY, cubaREST.restApiToken);
+      })
+      .catch(err => {
+        if (err.response) {
+          err.response.json().then((message) => {
+            if (message.error === 'invalid_grant') {
```
Comment (vadimbasko) [inline packages/front-generator/src/generators/react-native/app/template/components/Login.tsx]: I think we need separate file with error constants, with js doc description for each one.

File: packages/front-generator/src/generators/react-native/app/index.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      this.destinationPath(),
+      this.model,
+      undefined,
+      { globOptions: { dot: true } } // copy all files including hidden (will not copy hidden directories)
+    );
+
+    const sdkDest = 'cuba';
+    this.log(`Generating SDK model and services in ${sdkDest}`);
+
+    console.log(this.modelPath);
```
Comment (vadimbasko) [inline packages/front-generator/src/generators/react-native/app/index.ts]: Looks like console.log should be removed or replaced with this.log

File: packages/front-generator/src/generators/react-native/app/index.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
+import {BaseGenerator, readProjectModel} from '../../../common/base-generator';
```
Comment (vadimbasko) [inline packages/front-generator/src/generators/react-native/app/index.ts]: In this file there is code duplication with ReactTSAppGenerator described in react-typescript/app/index.ts. Refactoring needed, but may be in future issues.

File: package.json [hunk 4]
```diff
@@ -16,15 +16,20 @@
   "scripts": {
     "lerna": "lerna",
     "commit": "node_modules/commitizen/bin/git-cz",
-    "pack-client-libs": "lerna run prepublishOnly && lerna exec --scope @cuba-platform/re* 'npm pack -q'",
-    "update-client-libs": "npm run pack-client-libs && node scripts/update-client-libs.js",
-    "bootstrap-react-app": "node scripts/generate-react-client-scr.js && npm run update-client-libs",
+    "pack-react-client-libs": "lerna run prepublishOnly --scope '{@cuba-platform/frontend-generator,@cuba-platform/rest,@cuba-platform/react-core,@cuba-platform/react-ui}' && lerna exec --scope '{@cuba-platform/rest,@cuba-platform/react-core,@cuba-platform/react-ui}' 'npm pack -q'",
+    "pack-react-native-client-libs": "lerna run prepublishOnly --scope '{@cuba-platform/frontend-generator,@cuba-platform/rest,@cuba-platform/react-core}' && lerna exec --scope '{@cuba-platform/rest,@cuba-platform/react-core}' 'npm pack -q'",
```
Comment (vadimbasko) [inline package.json]: For long scripts I propose to use js files. Something like 
```
"pack-react-client-libs": "node scripts/pack-libs react-client"
"pack-react-native-client-libs": "node scripts/pack-libs react-native-client"
```

File: packages/front-generator/src/generators/react-native/app/index.ts (hunk 3 above)
Comment (vyacheslav-pushkin) [reply packages/front-generator/src/generators/react-native/app/index.ts]: Created a separate issue #86 for that as there is also code duplication between different React and Polymer generators

**Pull request merged**